iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/posix_call.inl

```diff
@@ -170,32 +137,46 @@ inline PosixCallEvaluator<ReturnType>::PosixCallEvaluator(internal::PosixCallDet
 
 template <typename ReturnType>
 template <typename... IgnoredErrnos>
-inline cxx::expected<PosixCallResult<ReturnType>, PosixCallResult<ReturnType>>
-PosixCallEvaluator<ReturnType>::evaluateWithIgnoredErrnos(const IgnoredErrnos... ignoredErrnos) const&& noexcept
+inline PosixCallEvaluator<ReturnType>
+PosixCallEvaluator<ReturnType>::ignoreErrnos(const IgnoredErrnos... ignoredErrnos) const&& noexcept
 {
     if (!m_details.hasSuccess)
     {
-        internal::AssignReturnValueIfItsErrno<ReturnType, std::is_convertible<ReturnType, int32_t>::value>::call(
```

Comment:
Is `AssignReturnValueIfItsErrno` used anywhere else or can it be removed?

Reply:
upsi I thought I already removed it